code/modules/power/apc.dm

```diff
@@ -581,6 +581,32 @@
 			return TRUE
 
 /obj/machinery/power/apc/attackby(obj/item/W, mob/living/user, params)
+	if(istype(W, /obj/item/kitchen))
+		var/metal = FALSE
+		var/shock_source = FALSE
```

Comment:
Again, why is this a boolean? You set it to an atom later on, you probably want null instead.